apps/backend/user/src/models/user.ts

```diff
@@ -0,0 +1,12 @@
+export interface User {
+  user_id: number;
+  username: string;
+  email: string;
+  password: string;
+  name: string;
+  bio: string;
+  profile_pic_url: string;
+  verified: boolean;
+  created_at: Date;
+  modified_at: Date;
+}
```

Comment:
Not sure if this is used in multiple places, but all the user endpoints have a slightly different return type